recipe/meta.yaml

```
@@ -1,42 +1,52 @@
-{% set version = "0.4.3" %}
+{% set version = "0.6.2" %}
 
 package:
   name: openapi-schema-validator
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/o/openapi-schema-validator/openapi_schema_validator-{{ version }}.tar.gz
-  sha256: 6940dba9f4906c97078fea6fd9d5a3a3384207db368c4e32f6af6abd7c5c560b
+  sha256: 11a95c9c9017912964e3e5f2545a5b11c3814880681fcacfb73b1759bb4f2804
 
 build:
+  skip: true # [py<38]
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
     - pip
     - poetry-core >=1.0.0
-    - python >=3.7
+    - python
   run:
-    - attrs >=19.2.0
-    - jsonschema >=4.0.0,<5.0.0
-    - python >=3.7
+    - python
+    - jsonschema >=4.19.1,<5.0.0a0
+    - jsonschema-specifications >=2023.5.2,<2024.0.0
     - rfc3339-validator
 
 test:
+  source_files:
+    - tests
   imports:
     - openapi_schema_validator
   commands:
     - pip check
+    # skipping test_array_prefixitems_invalid because of https://github.com/python-openapi/openapi-schema-validator/issues/153
+    - pytest -vv --cov=openapi_schema_validator --cov-branch --cov-report=term-missing:skip-covered --no-cov-on-fail --cov-fail-under=80 -k "not test_array_prefixitems_invalid[value0]"
   requires:
     - pip
+    - pytest-cov
```

```suggestion
    - pytest -v -k "not test_array_prefixitems_invalid[value0]"
  requires:
    - pip
    - pytest
```
We don't need to run tests with coverage.